core/src/banking_stage/transaction_scheduler/transaction_priority_id.rs

```diff
@@ -25,3 +29,15 @@ impl PartialOrd for TransactionPriorityId {
         Some(self.cmp(other))
     }
 }
+
+impl Hash for TransactionPriorityId {
+    fn hash<H: Hasher>(&self, state: &mut H) {
+        self.id.hash(state)
+    }
+}
+
+impl TopLevelId<Self> for TransactionPriorityId {
+    fn id(&self) -> Self {
+        *self
+    }
+}
```

Comment:
So this looks a bit odd, but there's a reason it's like this.

There's a custom prioritization function passed to the `PrioGraph`.
That function produces anything which impls TopLevelId<Id> where `Id` is the identifier for txs. That function **can** modify priority based on anything it likes...for example number of edges coming out of the node...but that is not currently being done, and it just uses the priority of this field directly for now.